cmake/install/userver-multi-index-lru-config.cmake

```diff
@@ -0,0 +1,11 @@
+include_guard(GLOBAL)
```

Comment:
Hi! Can you please add some documentation on the feature? The description says it is "from" boost - is it a copy-paste of implementation? Or maybe some adapted version of it? Or a wrapper?

Reply:
We'll add some docs later. Prototype with proper support of CMake goes first.

I'm discussing the PR with the author in Telegram